modules/standard/AutoMath.chpl

```diff
@@ -21,6 +21,12 @@
 /*
 This module provides mathematical constants and functions.
 
+.. warning::
```

Comment:
Does this suppress the warning from the attribute? Or are there now 2 unstable warnings in the docs?

Reply:
It suppresses the warning, since the note includes the word "unstable" in it directly.  I double checked the generated page to make sure the old one was gone :)

Reply:
(For AutoMath, it was getting inserted here: https://chapel-lang.org/docs/main/modules/standard/AutoMath.html#constant-and-function-definitions)